2
cuda
Traceback (most recent call last):
  File "train.py", line 435, in <module>
    main()      
  File "train.py", line 432, in main
    train(dataset, model_name)
  File "train.py", line 428, in train
    model = multi_mode(device, mu, cov, region_num = region_num, topic_num = topic_num, loc_num = loc_num).to(device)
  File "/home/wuning/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 381, in to
    return self._apply(convert)
  File "/home/wuning/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 187, in _apply
    module._apply(fn)
  File "/home/wuning/anaconda3/lib/python3.6/site-packages/torch/nn/modules/rnn.py", line 116, in _apply
    ret = super(RNNBase, self)._apply(fn)
  File "/home/wuning/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 193, in _apply
    param.data = fn(param.data)
  File "/home/wuning/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 379, in convert
    return t.to(device, dtype if t.is_floating_point() else None, non_blocking)
RuntimeError: CUDA out of memory. Tried to allocate 5.25 MiB (GPU 0; 10.92 GiB total capacity; 40.11 MiB already allocated; 8.50 MiB free; 19.00 KiB cached)
